README.md

```diff
@@ -522,7 +522,7 @@ Verify flags:
 Unimplemented `docker run` flags:
     `--attach`, `--blkio-weight-device`, `--cgroup-parent`, `--cpu-rt-*`, `--detach-keys`, `--device-*`,
     `--disable-content-trust`, `--domainname`, `--expose`, `--health-*`, `--ip6`, `--isolation`, `--no-healthcheck`,
-    `--oom-score-adj`, `--link*`, `--mac-address`, `--publish-all`, `--sig-proxy`, `--storage-opt`,
+    `--link*`, `--mac-address`, `--publish-all`, `--sig-proxy`, `--storage-opt`,
```

Comment:
I was not sure where should I update the documentation i.e under which section for run. Is it possible to suggest it?

Reply:
Right after (or before) `--oom-kill-disable` ?

Reply:
Its grouped under cgroups flag; the change itself is not a cgroup change; that is why I was not sure

Reply:
Maybe change "cgroup" to "resources"